src/section-graphing-equations.mbx

```diff
@@ -17,6 +17,8 @@
 
     <title>Graphing Equations</title>
 
+    <!-- Why graph equations? -->
```

Comment:
Ralf, is this a stray comment? If it should be here, what do you mean by this? Is it a note to add a motivational subsection? Or is it a note to coauthors asking why have this section at all?

I want to not leave ambiguous comments that some future reader (including you and me) have a tough time understanding what was the intent.